src/components/Icons/CollectionRowActionIcon.js

```diff
@@ -41,22 +47,28 @@ const CollectionRowActionIcon = ({
       ActionSheetIOS.showActionSheetWithOptions({
         options: [
           'Cancel',
+          'Edit Collection',
+
           CollectionsDialogText[COLLECTIONS_DIALOG_ACTIONS.RENAME].title,
           CollectionsDialogText[COLLECTIONS_DIALOG_ACTIONS.DUPLICATE].title,
```

Comment:
This may be a Drashko question, but does Duplicating a Collection with the simple Modal make sense anymore? Wouldn't the user want to go to the `CollectionInputScreen`, edit the name there and change settings, then hit `Duplicate`?